benchexec/tools/divine4.py

```diff
@@ -86,16 +94,15 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         and should give some indication of the failure reason
         (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
         """
-
-        if not output:
+        if not run.output:
             return "ERROR - no output"
 
-        last = output[-1]
+        last = run.output[-1]
 
-        if isTimeout:
+        if run.was_timeout:
             return "TIMEOUT"
 
-        if returncode != 0:
+        if run.exit_code.value != 0:
```

Comment:
It seems this needs the same handling as for `divine`.

Reply:
fixed in 59886567a1e060133a94b09ebc336b10a5336361